.travis.yml

```diff
@@ -22,13 +27,13 @@ env:
-    - secure: "f24fzdhSzxlNIYtCmYNWfcM4zFZlzNEkOWGfRqJh3Jurz9SpwSXIWwTlXa6J+1FAe+AS7yHdp5TqXmzYm9LIHOXWxsAhOYSC0ueADrTZqslWN6RQ4Skr0tHZm/4MAJfSIWYUVsllo78u1KiTEaOVsnu6yJWdcwr6sA1HSvid2myn4SUqSsm2GNry0Yqb2qBUj6y7YBWUSZdusl2jERVUhMPpvr5gSquZC8H0OYDF/5OCuk8XgLwBPPFEHT3dezx0fusQaDajYag05LI5NO6F8W+IOk36speRHkdfjPyGgS4PTp0pMhRy5WCpXXQ/80JiCSXp+4U2a3IunaisckDkJrSuI3zk9Xc8KLITYX2kH58K9VMMkw9CGpXOWExqIAXaQBFSMnVaIEHiTzWs3IumxWEzfCswuXoc44wSRMzqwZnwA10QaWdGHrOpGmaaKdmST15NbgqLI313vd37k4Hp08FVD4m2CzgoMlqC53DJV8c6k3IfOc875fwEQWGgLtZ2FrLC0UW/MJoWq238ss4T12pAZF4NkIRxMpUV0HfRW1hpMio02qk7qfUpWVVclg/zGqxGTG9ctbU31sVvolMmtwd9AR2tMmSmyTAonlgA8vxGathYSXdgLLpbXbWRFWOFHOZGJpdGCaDc42NhPkaobpguntKj8DzNmaKf3l16OtA=" #driver=testingbot
-    - secure: "dhaXaeTTLVOxkBfCvt/+yPlP+hOKvdWHCXE7sbDDsCIj/v5WEK724LUalMbDfJHolndnKqSN4fccjmCMIPeuioOIGt54kHmz3atq9PZuegYu6qVSuLr1ugANw3V99a6YDxZLdruL0JHHCwuTxLYLoWT//u8y4tMsbUZQokNel3mqQx3o/x7OrPKVUg0V5i8Myb8yBtV+9SKw1wQaK3U5jR4e9ze7bhINlJSUaBEe1t55KSptsEfnhwL+LY76i+tMYDrl5EcrjK/8Y6gsHZAocXJYuu5MNeHdg4ufwSafO9vm9fssWhQvs+TkHrtxDwc8qgXQT4X1UMZyeRuZFYKd8LfuaCikL3+FAx/9MmbLXv5RaLZBxlDtw5MglmDaSRGzLEfuhADi0fWfmaxHGQjEU3BTWKiFtesHO1i7eA2xwhMoj0Vakpl2rBCVp0ckZ9vxhEO8gEZulYBU6xb968DNO4d5Qvh5WOBu9F3ftu+7p5x5lbPtsa3HYtslPaCpvov6qLOroeOJBcDJ3yNmq1d4gkOgH7QlYAWVTuBSz7UY25nNL6LM+fbc2jOev1/JrQ4li3cSNx38x5jF3FNQKEJBkFibyiwGAe24zypaMafswTpMGLbOeBGx60U+nRkXPw1FfT8zwiqNI5Phttd1zHdS7J6MEMEB2tCwXlegQffVgjo=" #driver=browserstack
-    - secure: "XRQ04Igl9AWAYrMxHJB93BPPAT0ZHMdFeSwMjb4LlSRujU3/Rmp3f/ZgGYonChMs137MqGcraG722DSdbQs5NFthaL3iIl997Hdke6f/41XYYnn8UiI3QU4f1h4f7umr0cDOn+ewdh/TGuKaRWCEYkmT2VAqszepcGa8wnhcoy21YxlKC/AsQOfOeCHYa/Y/yhHVSVW3lxwg8g7wZswfabhR7ITKGXJnvvrFv2+chh+pW+BDo3cs+zQod0/ETYDQvA8ZO5imgx7CsQiahK6fsOK9rWlfCcYPC6Hn4SMm4VJ3Ezlm9MUfd7vyziHei9uZHgs7hPBURwsvEHJ8Rt0rxCzmeDz9ihFDbs9d1RDfEKFtpAuZ5MGrqQPm6555VLn0RmIb5NxBf8/J+Cq/0iKAeJLKStjTGjwS9jpWE1bAhTKJCUwM7ATN9qhmKosu2uYL33ZsA4+54D8YsHZ8ucA/p/NZDXOPFh1tNzz4/9TlWL0Jl7dHoxdvaxuyoc6xHKzvLqcCvxLRLJFlePJaZzLxitSB7u4jtIlUeR4gRki54XL20lQb6mCi5JQN5lnaTX4g//KVl82N6QvmI54Zxd6jftJ+IiPDfkTId3gzFR53Sy9SUJoa3OgjPxOTJSDFp/I4lv1oIxxbBnZCBx6aN2rYlG3OoUSQtke3/6DKG+CFJm8=" #driver=crossbrowsertesting
+    - secure: "k/2zzUeePe753UyPxCqCiiIwafhO19V0U4NOo1IToZUYwsGQ8L3uzzpNU8PfJDYkPYWTZBnvQNhm3gJVgvGvsVpzawmWVUHtkX+1bL8+dpVnqBjrlnR3+Du6OsLT7PtGGWOFuZDG3XDpQumI2vLq1osDZsnYB9Y+5cDY1Siu4GMY2G/jEOW5I8khufo4ubDhsY5vEmz1T+7N7G4lmyQy3Io3Jh3Ciws1Cku97ZbBT3RB+sPMlbAy40b0zZEZr0BuSogEUzfJdEt8nPQhd2nregwSkaeN9NgFVWl7nFdmBFTxr5wf86sUHH7vEd40k4uKx8BOYdtVcFAMWK/QmSXIQwO1WCHt1Fi2MloIIzGb7FkDuIS6cVbUm1GCtEArYHhX+Eti5Me/3j1hCGVtUwzNUv4PjocJ7pwy+H0DqRRwvTf77dUVVzBKMHcXmm8MMMu/AT0iTQTfl3R4JX3WgtSnxzLUAoa48K2gXWFqi81lwSG7C0lTvPG6RPF5guY+MgC1lNXVRRV5+OfnPgStwCi9/N7JLddLvmo8S8luAK8bAPcAxxQ7mX38/pQvC1BXWq7EMSJhIP6vTLz2T5foDv8vCVAg6x7H4hnbNzCu2qBIl9o55wcbIQTC0ACaEeDOdI7hHRVTluKIfMqnWr0si9k4CK45tiktZF9/bGjKQGQxH5A=" #driver=poltergeist
```

Comment:
Why do you secure not secured information here? Have you introduced additional settings?

Reply:
Yes, I've added private variables.